notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Now that the default repr only shows the first 5 and last 5 rows, you actually get head and tail in one go .. </p><p>So wondering if we should just show that first, still saying afterwards (or showing one of the two) that you can use those for more specifically seeing first or last n rows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Adjusted as such: using default repr first, introduce head and add tail as a note.</p>